frontend/src/App.jsx

```diff
@@ -14,4 +16,12 @@ function App() {
   );
 }
 
-export default registerRootComponent(App);
+function reduxApp() {
+  return (
+    <Provider store={store}>
```

Comment:
why haven't you put the provider in the original App component like this? isn't the 2 component unnecessary? seems overcomplicated to me. I think it's more simple:
what do you think?
```suggestion
function App() {
  return (
    <Provider store={store}>
      <View style={commonStyles.container}>
        <Menu />
        <Troop />
      </View>
    </Provider>
  );
}
```

Reply:
I saw you changed it, but the separated way was OK for me as well, I just wanted to know your reasons behind it:)